web/index.html

```diff
@@ -28,8 +28,9 @@
 
```

Comment:
Remove all the web native splash screen related files
since it is not required.